app/src/main/res/layout/fragment_example2.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:gravity="center"
     android:paddingBottom="@dimen/activity_vertical_margin"
     android:paddingLeft="@dimen/activity_horizontal_margin"
     android:paddingRight="@dimen/activity_horizontal_margin"
     android:paddingTop="@dimen/activity_vertical_margin"
-    tools:context="io.intrepid.skeleton.MainActivity"
     >
 
     <TextView
+        android:id="@+id/example2_text"
```

Comment:
This is heavily depend on how the designs turn out. I don't think our designed follows the same guidelines between different projects, so I don't want to impose any guidelines here.